superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -205,6 +203,14 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
 
       fetchResource(id)
         .then(() => {
+          // Altering 'extra' JSON before returning it to 'extra' text field
+          if (dbFetched && dbFetched.extra) {
```

Comment:
dbFetched?.extra

Reply:
No longer parsing JSON so I'm marking this as resolved.

Reply:
Parsing JSON after all, fix is in this [commit](https://github.com/apache/superset/pull/14293/commits/a10aa1b87ab5b85dd999c4ec6fdc78a14f83ca4f)